backbone/src/main/java/org/researchstack/backbone/ui/step/body/FormBody.java

```diff
@@ -96,4 +102,14 @@ private StepBody createStepBody(QuestionStep questionStep) {
             throw new RuntimeException(e);
         }
     }
+
+    private void hideSoftKeyboard() {
```

Comment:
we could send the view group as parameter here, so we don't need to hold the reference to the parent in the class, at least we need it for something else, but It doesn't seems to

Reply:
Nicely done. I missed you :)
I also added a null check.

Reply:
haha. Well, I'm not familiarized with the ImputMethodManager, but this seems to be a method that could be added as a public static method in a utils class?

Reply:
Ja! There was already a method in ViewUtils...though it doesn't work.
I added our in there too with a different signature. It works nicely.